compiler/src/Language/Bond/Codegen/Cpp/Types_h.hs

```diff
@@ -30,8 +30,9 @@ types_h :: [String]     -- ^ list of optional header files to be @#include@'ed b
         -> Bool         -- ^ 'True' to generate enum definitions into a separate file /base_name/_enum.h
         -> Maybe String -- ^ optional custom allocator to be used in the generated code
         -> Bool         -- ^ 'True' to generate constructors with allocator
+        -> Bool         -- ^ 'True' to generate type aliases
```

Comment:
I believe this is an API change and we have to bump gbc library

Reply:
There is already `(minor bump already done in bond.cabal)` note in changelog.

Reply:
This is a breaking change unless you also add a function that takes one fewer parameter and delegates to this one. (A breaking change in the Bond Haskell library is fine for a change like this.)

Reply:
Looks like that is already done. I'll fix the changelog note to `major bump`.